app/uk/gov/hmrc/pdsauthcheckerstub/services/ValidateCustomsAuthService.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.pdsauthcheckerstub.services
+
+import uk.gov.hmrc.pdsauthcheckerstub.models.{AuthType, Eori, PdsAuthResponse, PdsAuthResponseResult}
+
+import java.time.LocalDate
+import javax.inject.Singleton
+
+@Singleton()
+class ValidateCustomsAuthService() {
```

Comment:
again, weird name, doesn't really describe what it does.

Reply:
Maybe a small converter class that takes in an eori and spits out a result  could be useful here, a single place to put all the test data generation rules and test them under a single spec. (ResultBuilder maybe?)